src/lib.rs

```diff
@@ -1,13 +1,18 @@
 //! # Wrappers for compatibility with rand 0.4
 
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(feature = "std")]
```

Comment:
`PhantomData` is also available in `core`, we could just use that here and lose the compiler directive, I *think*.

Would that make it feasible to offer these wrappers for std and non-std alike?

Reply:
There's different conventions on how to handle the std vs non-standard variants, either importing everything from core or doing something like https://docs.rs/rand/0.4.3/src/rand/lib.rs.html#250. I have no idea which one is *en vogue* at the moment.

Reply:
You're right, thanks! I amended the commit.